README.md

```diff
@@ -1,24 +1,33 @@
 # CarLens
+![blog post image 1440x500 1](https://user-images.githubusercontent.com/18245585/51602699-ea70bd80-1f07-11e9-9bf9-8e333ffb63da.png)
 
-Welcome to the **CarLens** project. It's an internal Netguru's application, which uses Machine Learning and Augumented Reality technologies. Its main aim is **car recognition**. The application uses Core ML, ARKit, Vision, and Lottie frameworks. 
+[<img src="https://user-images.githubusercontent.com/18245585/51603579-771c7b00-1f0a-11e9-866d-5ce9fb0d1aca.png" width="150">](https://itunes.apple.com/us/app/carlens/id1417168518?mt=8)
```

Comment:
Shouldn't this hero image have an iPhone in it? 😄

Reply:
Also, why are you using HTML `<img>` tags everywhere instead of Markdown for inserting images?

Reply:
🤦‍♀️I did check whether it has an iPhone in it on another image. I guess we should ask designers to update it, cause no iPhone analog is available.

As for markdown: it kind of didn't work with setting width like `| width = 50)` with having an URL at the same time. Maybe I didn't it in a wrong way? For others, I'll just automatically copied them from this one, to be honest.

Reply:
I guess that markdown just is no longer supported https://github.com/jgm/pandoc/issues/2554#issuecomment-158936360
You can see GH didn't convert that parts to an image

Reply:
As for an image: I've submitted a task for design team :)

Reply:
Why do you need to explicitly set widths?

Reply:
Cause one of them is shorter than another and it doesn't look nice.
For download image: it's too big (like the whole screen width).

Reply:
Alright. For the record, if you're displaying an image at some fixed width anyway, I recommend first uploading images with correct dimensions and then using them without `width` tag. You can spare unnecessary data transfer this way. :)

Reply:
That's just a great note!